tests/testsrc/test262.properties

```diff
@@ -3186,6 +3186,206 @@ language/expressions/arrow-function 209/333 (62.76%)
     scope-paramsbody-var-close.js
     scope-paramsbody-var-open.js
 
+language/expressions/assignment 200/468 (42.74%)
```

Comment:
I'm trying to refresh my memory on test262.properties -- is this a long list of tests that we now have to disable becuase we broke them, or were these tests never run and we're actually able to pass 58% of them?

In other words, do these test262 changes represent an improvement, or a regression?

Reply:
The test262Suite only looks at tests in directories that are already included in trst262.properties

So what you're seeing here in inclusion of a new directory full of tests, some of which already pass

Ran into this recently, tried to create a PR that will include ALL directories automatically, because this is a gap in our 'security net'. Didn't get around to finish it though....

Reply:
f772166a921a842a7ce86429f39b37319959be1c First I added `language/expressions/assignment` tests to test262.properties in the first commit.
1fd026f0207ac54ac7c40bdcdd60c6501f16589b And two of those tests were improved.

Reply:
I added all directories to test262.properties. ref. #1531
I would like to merge that PR first.